fake-koji/src/main/java/org/fakekoji/jobmanager/model/PlatformConfig.java

```diff
@@ -48,4 +48,13 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id, tasks, provider);
     }
+
+    @Override
+    public String toString() {
+        return "PlatformConfig{" +
+                "id='" + id + '\'' +
+                ", tasks=" + tasks +
+                ", provider='" + provider + '\'' +
+                '}';
+    }
```

Comment:
I was recently adding soime toString but look like we do not have exact clash. Stil please be carefull with merging those

Reply:
The only clash was in OToolVariable (I kept your version), other toStrings should be fine

Reply:
TY! few more around i'm afraid:( Sorry